src/main/scala/beam/router/BeamSkimmer.scala

```diff
@@ -57,12 +57,14 @@ class BeamSkimmer @Inject()(val beamConfig: BeamConfig, val beamServices: BeamSe
   private def initialPreviousSkims(): TrieMap[(Int, BeamMode, Id[TAZ], Id[TAZ]), SkimInternal] = {
     if (beamConfig.beam.warmStart.enabled) {
       try {
-        skimsFilePath
+        val previousSkims = skimsFilePath
           .map(BeamSkimmer.readCsvFile)
           .getOrElse(TrieMap.empty)
+        logger.info(s"Previous skims successfully loaded from path '${skimsFilePath.getOrElse(",")}'")
```

Comment:
Why is the `getOrElse` a comma? Did you mean to do a `mkString(",")`?

Reply:
could be empty string .. 
`getOrElse("")`

Reply:
Maybe be more explicit and make it something like "Not Found"?